sqlglot/dialects/presto.py

```diff
@@ -173,6 +173,37 @@ def _unix_to_time_sql(self: Presto.Generator, expression: exp.UnixToTime) -> str
     return f"FROM_UNIXTIME(CAST({timestamp} AS DOUBLE) / POW(10, {scale}))"
 
 
+def _jsonextract_sql(self: Presto.Generator, expression: exp.JSONExtract) -> str:
+    is_json_extract = self.dialect.settings.get("variant_extract_is_json_extract", True)
+
+    if isinstance(is_json_extract, str):
+        # if the kwarg was set by the user it's stored in a string
+        is_json_extract = is_json_extract.lower() == "true"
```

Comment:
i don't think we need lower

Reply:
Just tested this again, `is_json_extract` is exactly what the user has passed in the dialect settings so it's case sensitive afaict. 

Was having a [https://github.com/tobymao/sqlglot/pull/3725#pullrequestreview-2153477185](discussion) with George regarding this, would appreciate your input